src/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/Mocks/IActiveConfiguredProjectProviderFactory.cs

```diff
@@ -18,14 +18,14 @@ public static IActiveConfiguredProjectProvider ImplementActiveProjectConfigurati
         }
 
         public static IActiveConfiguredProjectProvider Create(
-            Func<ProjectConfiguration>? getActiveProjectConfiguration = null,
-            Func<ConfiguredProject>? getActiveConfiguredProject = null)
+            Func<ProjectConfiguration?>? getActiveProjectConfiguration = null,
+            Func<ConfiguredProject?>? getActiveConfiguredProject = null)
         {
             var mock = new Mock<IActiveConfiguredProjectProvider>();
-            mock.SetupGet(p => p.ActiveProjectConfiguration)
+            mock.SetupGet<ProjectConfiguration?>(p => p.ActiveProjectConfiguration)
```

Comment:
Why was this needed?

Reply:
You'll see several of the mock setup methods annotated in this way.

The problem is that the interfaces they're mocking aren't annotated, so we have to tell the compiler when mocked members have some kind of nullability.

In this particular case, a caller of `Create` was using `!` on the `getActiveProjectConfiguration` argument because its `Func` returned a nullable value. The docs say `ActiveProjectConfiguration` can be null, so I updated this mock construction to reflect that.

If `IActiveConfiguredProjectProvider` was annotated, the type argument here would be redundant.